x/programs/rust/wasmlanche-sdk/src/types.rs

```diff
@@ -10,10 +13,15 @@ pub type Gas = u64;
 /// A struct that enforces a fixed length of 32 bytes which represents an address.
 #[cfg_attr(feature = "debug", derive(Debug))]
 #[derive(Clone, Copy, PartialEq, Eq, BorshSerialize, BorshDeserialize, Hash)]
-pub struct Address([u8; Self::LEN]);
+#[repr(transparent)]
+pub struct Address([u8; 33]);
+
+// # Safety: Pod is safe to implement for arrays of bytes
```

Comment:
There's an implementation for 32, but not 33, because they can't implement `Pod` for every single array type.